### PR TITLE
AEA-1276 Fix/tests

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

fix for tests

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/helpers/multiple_executor.py [hunk 1]
```diff
@@ -197,12 +196,17 @@ def _start_tasks(self) -> None:
...
             self._future_task[future] = task
 
-    async def _wait_tasks_complete(self, skip_exceptions: bool = False) -> None:
+    async def _wait_tasks_complete(
+        self, skip_exceptions: bool = False, on_stop: bool = False
+    ) -> None:
         """
         Wait tasks execution to complete.
 
         :param skip_exceptions: skip exceptions if raised in tasks
```
Comment (DavidMinarsch) [inline aea/helpers/multiple_executor.py:205]: missing param in docstring

Comment (DavidMinarsch) [reply aea/helpers/multiple_executor.py:205]: where do we set `is_running` to False?

File: tests/test_packages/test_connections/test_p2p_libp2p/test_fault_tolerance.py [hunk 2]
```diff
@@ -258,6 +258,7 @@ def test_envelope_routed_from_client_after_relay_restart(self):
         self.multiplexer_relay.disconnect()
         self.change_state_and_wait(self.multiplexer_relay, expected_is_connected=False)
 
+        print(1111111111111, "DISCONNECTED", flush=True)
```
Comment (DavidMinarsch) [inline tests/test_packages/test_connections/test_p2p_libp2p/test_fault_tolerance.py:261]: remove

File: aea/helpers/multiple_executor.py (hunk 1 above)
Comment (solarw) [reply aea/helpers/multiple_executor.py:205]: in stop method

File: tests/test_packages/test_connections/test_p2p_libp2p/test_fault_tolerance.py (hunk 2 above)
Comment (solarw) [reply tests/test_packages/test_connections/test_p2p_libp2p/test_fault_tolerance.py:261]: done

File: aea/helpers/async_utils.py [hunk 3]
```diff
@@ -361,73 +360,6 @@ def stop(self) -> None:
         _default_logger.debug("Stopped.")
 
 
-class AwaitableProc:
```
Comment (DavidMinarsch) [inline aea/helpers/async_utils.py:364]: Why removed?

File: aea/helpers/multiple_executor.py (hunk 1 above)
Comment (DavidMinarsch) [reply aea/helpers/multiple_executor.py:205]: docstring incomplete

File: aea/helpers/async_utils.py (hunk 3 above)
Comment (solarw) [reply aea/helpers/async_utils.py:364]: not used at all, some time ago i replaced this class with std library solution

File: aea/helpers/multiple_executor.py (hunk 1 above)
Comment (solarw) [reply aea/helpers/multiple_executor.py:205]: done

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**